<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Categoria</class>
 <widget class="QDialog" name="Categoria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    font-family: Arial, sans-serif;
    background-color: #f5f5f5;  /* Light gray background for the form */
    color: #333;                /* Dark text color */
}

QLabel {
    font-weight: bold;          /* Bold labels for field names */
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Categorías</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>315</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Visualización</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>369</y>
     <width>656</width>
     <height>192</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>656</width>
     <height>0</height>
    </size>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QWidget" name="">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding:8px 0px;</string>
      </property>
      <property name="text">
       <string>Ingrese el nombre de la categoría</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nombreLineEdit">
      <property name="styleSheet">
       <string notr="true"> border-radius: 5px;       
   	border: 1px solid black;
    padding: 4px;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="registrarBtn">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #3498db;     
    color: white;                  
    border-radius: 5px;       
   	border: 1px solid black;
    padding: 4px 18px;
}
QPushButton:hover {
    background-color: #2980b9;
    border: 1px solid #ffffff;
}
QPushButton:pressed {
    background-color: #1c6692;
}</string>
      </property>
      <property name="text">
       <string>Registrar Categoría</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="eliminarBtn">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #3498db;     
    color: white;                  
    border-radius: 5px;       
   	border: 1px solid black;
    padding: 4px 18px;
}
QPushButton:hover {
    background-color: #2980b9;
    border: 1px solid #ffffff;
}
QPushButton:pressed {
    background-color: #1c6692;
}</string>
      </property>
      <property name="text">
       <string>Eliminar Categoría</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="actualizarBtn">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #3498db;     
    color: white;                  
    border-radius: 5px;       
   	border: 1px solid black;
    padding: 4px 18px;
}
QPushButton:hover {
    background-color: #2980b9;
    border: 1px solid #ffffff;
}
QPushButton:pressed {
    background-color: #1c6692;
}</string>
      </property>
      <property name="text">
       <string>Actualizar Categoría</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="printBtn">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #3498db;     
    color: white;                  
    border-radius: 5px;       
   	border: 1px solid black;
    padding: 4px 18px;
}
QPushButton:hover {
    background-color: #2980b9;
    border: 1px solid #ffffff;
}
QPushButton:pressed {
    background-color: #1c6692;
}</string>
      </property>
      <property name="text">
       <string>Imprimir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>149</y>
     <width>195</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding:8px 0px;</string>
      </property>
      <property name="text">
       <string>Ingrese una descripción</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="descLineEdit">
      <property name="styleSheet">
       <string notr="true"> border-radius: 5px;       
   	border: 1px solid black;
    padding: 4px;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
